### DISCUSSION TITLE
Observability enhancement

### DISCUSSION THREAD

**Discussion by xxm404** [apache/apisix#5554] [Ideas]
APISIX support health check for upstreams, why not expose healthy/unhealthy nodes metrics ? like AWS ELB, support many useful metirc https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-cloudwatch-metrics.html

**Comment by tokers** [comment]
Sounds good, do you have some specific proposals?

**Reply by xxm404** [reply]
Maybe APISIX need to expose healthy nodes metrics by Prometheus format when opening the upstream health check(same for other metrics) and also monitor metrics should be set default opening, the API gateway need more monitor metrics

**Comment by xxm404** [comment]
For target(nodes) health/unhealthy metrics, we can make some counters to record health nodes or unhealthy nodes, when a health check request failed we should increase the unhealthy counter and reduce the health counter.

**Reply by tokers** [reply]
Would you like to write a proposal to the dev@apisix.apache.org?

**Reply by xxm404** [reply]
how should I do?

**Reply by juzhiyuan** [reply]
Here is the Mailing list subscription guide, see https://apisix.apache.org/docs/general/subscribe-guide

After you subscribe the mailing list, kindly send a mail with title `[DISCUSS] XXX` to `dev@apisix.apache.org`

**Reply by juzhiyuan** [reply]
e.g https://lists.apache.org/list.html?dev@apisix.apache.org

**Reply by xxm404** [reply]
Thanks, I will send mail to `dev@apisix.apache.org`
